portfolio.php

```diff
@@ -26,16 +24,18 @@
                 <a href="index.html">Home</a>
             </nav>
             <div class="works">
-                <h3>
-                    Let me please introduce you my works.
-                </h3>
-                    <img class="demo" src="images/lamp1.png" alt="lamps">
-                    <img class="demo" src="images/lamp2.png" alt="lamps">
-                    <img class="demo" src="images/lamp3.png" alt="lamps">
+                <?php
+                echo '<h3>My projects:</h3>';
```

Comment:
if your not using a php variable, you dont need to put your HTML inside PHP. just put it as normal HTML outside the PHP tags